en/mkdocs.yml

```diff
@@ -2054,6 +2055,7 @@ plugins:
               'api-analytics/viewing/analytics-pages-latency.md': 'https://apim.docs.wso2.com/en/4.0.0/api-analytics/viewing/view-api-analytics-on-latency/'
               'api-analytics/viewing/analytics-pages-devices.md': 'https://apim.docs.wso2.com/en/4.0.0/api-analytics/viewing/view-analytics-on-devices/'
               'api-analytics/viewing/analytics-pages-report.md': 'https://apim.docs.wso2.com/en/4.0.0/api-analytics/viewing/download-api-analytics-reports/'
+              'api-analytics/samples/publishing-analytics-events-to-external-systems.md': 'https://apim.docs.wso2.com/en/4.0.0/api-analytics/samples/publishing-analytics-events-to-external-systems/'
```

Comment:
Why did you add a redirection here? This is only needed if the page name or path changes